### DISCUSSION TITLE
https://thanhle.blog/blog/tai-sao-minh-tin-vao-blockchain

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#23] [Ideas]
# https://thanhle.blog/blog/tai-sao-minh-tin-vao-blockchain

https://thanhle.blog/blog/tai-sao-minh-tin-vao-blockchain

**Comment by philip-taiwan** [comment]
Mình nghĩ blockchain sẽ không thành công khi áp dụng vào thị trường tài chính. Mình nghĩ blockchain phù hợp để giải quyết bài toán quản lý, truy xuất nguồn gốc, đặt chỗ của doanh nghiệp VD: hệ thống truy xuất nguồn gốc thực phẩm organic cho người sống healthy, áp dụng để tracking hàng hoá trong ngành logistics, đặt chỗ trước trong hệ thống các dịch vụ công, bốc số ở bệnh viện, ...

**Reply by thanhlmm** [reply]
Mình thì nghĩ ngược lài vì việc truy xuất nguồn gốc ngay cả mô hình truyền thống đã không hoạt động hiểu quả rồi, thì có bỏ lên blockchain thì cũng vậy thôi. Blockchain có thể lưu lại mọi thứ mà không thể thay đổi được, nhưng ai đảm bảo data lưu lên blockchain là đúng?

**Reply by phonganh841989** [reply]
Đúng vậy, mình cũng nghĩ như Thành. cảm ơn bài viết và góc nhìn của bạn.
